content/articles/automation_lessons.md

```diff
@@ -0,0 +1,110 @@
+title: Automation Lessons
+tags: software engineering, collaboration
+author: cdent
+created: 2025-02-26
+
+```intro
+I initially wrote the following for work colleagues, but sharing more widely
+in case it might apply elsewhere.
+
+A lesson from long ago that has stuck with me and been proven valuable over and
+over:
+
+> When creating software, ensure there are working harnesses for testing,
+> installation, and upgrade first.
+```
+
+Over the years as development technologies have matured the lesson has evolved
```

Comment:
```suggestion
Over the years as development technologies have matured, that lesson has evolved
```

The comma significantly improves readability IMO. I'm less sure whether `s/the/that/` is sensible.